Source/Blazorise.Bootstrap/Styles/utilities/_display.scss

```diff
@@ -0,0 +1,14 @@
+﻿@each $breakpoint in map-keys($grid-breakpoints) {
+    // only generate xxl that is not available in native Bootstrap 4
+    @if $breakpoint == "xxl" {
```

Comment:
does it even make sense to iterate?

Reply:
Maybe not. But I want to make it future proof if we introduce another breakpoint.

Reply:
Wouldn't future proof be an iteration **over only the keys that you need** instead of doing **ifs**?

Reply:
It has to be an `if` because `breakpoint-infix` has special rules where it checks if there are smaller sizes than are currently running.